src/reference/asciidoc/messaging-endpoints.adoc

```diff
@@ -1,22 +1,13 @@
 [[messaging-endpoints-chapter]]
 == Messaging Endpoints
 
-// BE SURE TO PRECEDE ALL include:: with a blank line - see https://asciidoctor.org/docs/user-manual/#include-partitioning
 include::./endpoint.adoc[]
-
 include::./gateway.adoc[]
-
 include::./service-activator.adoc[]
-
 include::./delayer.adoc[]
-
 include::./scripting.adoc[]
-
 include::./groovy.adoc[]
-
 include::./handler-advice.adoc[]
-
 include::./logging-adapter.adoc[]
-
 include::./functions-support.adoc[]
-// BE SURE TO PRECEDE ALL include:: with a blank line - see https://asciidoctor.org/docs/user-manual/#include-partitioning
```

Comment:
See that URL

>Note the empty lines before and after the include directives. This practice is recommended whenever including AsciiDoc content to avoid unexpected results (e.g., a section title getting interpreted as a line at the end of a previous paragraph).